cfeScraping/index.js

```diff
@@ -45,10 +46,49 @@ ScrapPage.prototype.clickButton = async function (id, time) {
     await this.page.click(id);
     console.log('Searching...');
     await this.page.waitForTimeout(time);
-    return console.log('Search succesfull');
+    return console.log('Search successful');
   } catch (err) {
     return console.error('Error: ', err);
   }
 };
 
+// eslint-disable-next-line space-before-function-paren
+ScrapPage.prototype.getDataTable = async function () {
+  try {
+    const data = await this.page.evaluate(
+      (selector = 'table.k-selectable tr') => {
```

Comment:
The selector should come from param, & apply template literals `${yorvar} tr` to get all the rows of the data. remember we want use in other tables